config/deploy/staging.rb

```diff
@@ -1,7 +1,8 @@
-set :application, 'joegattnet_v3_staging'
-set :branch,      'staging'
-set :deploy_to,   '/home/deployer/apps/joegattnet_v3_staging'
-set :rails_env,   'staging'
+set :application,           'joegattnet_v3_staging'
+set :branch,                'staging'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.